scripts/checkpoint_converters/convert_zarr_to_torch_dist.py

```diff
@@ -167,7 +172,7 @@ def convert(local_rank, rank, world_size, args):
         )
 
     with open_dict(model.cfg):
-        model.cfg.torch_distributed_checkpoint = True
+        model.cfg.dist_ckpt_format = 'torch_dist'
```

Comment:
Do you plan to make it configurable as discussed on slack in some thread?
I think it's an easy change and could be beneficial in some cases.